students/jon_cracolici/lesson01/assignment/test_unit.py

```diff
@@ -0,0 +1,221 @@
+"""Unit Test Module for Inventory Management Software."""
+
+import io
+import os
+import sys
+DIR_PATH = os.path.dirname(os.path.realpath(__file__))
```

Comment:
This one should instead be:

```
DIR_PATH = os.path.dirname(os.path.realpath(__file__)) + "/inventory_management"
sys.path.append(DIR_PATH)
```

This will allow the other classes to import directly from _inventory_management_ without having to add _inventory_management_ to the import statement.